evap/evaluation/templates/email_base.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+  <title>EvaP</title>
+  <style>
+    @media (max-width: 768px) {
+      .spacer {
+        width: 0px;
+      }
+
+      .content {
+        width: 100%;
+      }
+    }
+
+    @media (min-width: 768px) {
+      .spacer {
+        width: 15%;
+      }
+
+      .content {
+        width: 70%;
+      }
+    }
+  </style>
+</head>
+
+<body style="margin: 0px; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; font-size: 1rem; font-weight: 400; line-height: 1.5; color: #212529; background-color: #ffffff;">
+  <table style="background-color: #495057; width: 100%; border-collapse: collapse;" cellspacing="0" cellpadding="0">
+    <tr>
+      <td style="width: 25%;">
+        <a href="https://evap.hpi.de">
+          <img src="https://evap.hpi.de/static/images/evap.png" width="100%" style="height: auto; display: block;"/>
```

Comment:
I read your paragraph about the image never scaling the way you wanted it to. I'm fine with any solution that (a) works with modern & widespread email clients and (b) is valid HTML / CSS. It would be very interesting to know how other big companies circumvent the problems that some email clients seem to have here.

The `width="100%"` tag is not valid html, as [per standard](https://html.spec.whatwg.org/#attr-dim-width), you can only give a number ("non-negative integer") there, which will then be interpreted as CSS pixels. Have you tested out the HTML5 `sizes` attribute? Maybe that is used in most clients and gives you the flexibility that you need.

Reply:
While trying to understand how all of this works I looked into some html emails of some companies. But other than hard-coded image sizes I was not able to find anything special in the auto-generated templates with a ton of lines.
Sadly, the sizes attribute didnt help but I probably got it working with the width and a hard-coded pixel value. But I still have to test it.